### PR TITLE
add OmniFlix Hub asset list

### PR DESCRIPTION
**Pull request by harish551**

### CODE REVIEW

File: omniflixhub/assetlist.json [hunk 1]
```diff
@@ -29,6 +29,243 @@
...
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "agoric",
+            "base_denom": "uist",
+            "channel_id": "channel-58"
+          },
+          "chain": {
+            "channel_id": "channel-30",
```
Comment (JeremyParish69) [inline omniflixhub/assetlist.json:157]: Please register this IBC connection in the /_IBC/ directory

File: omniflixhub/assetlist.json [hunk 2]
```diff
@@ -29,6 +29,243 @@
...
+      "name": "USD Coin (Axelar)",
+      "display": "usdc",
+      "symbol": "USDC.axl",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "axelar",
+            "base_denom": "uusdc",
+            "channel_id": "channel-77"
```
Comment (JeremyParish69) [inline omniflixhub/assetlist.json:202]: Please register this IBC connection in the /_IBC/ directory

File: omniflixhub/assetlist.json [hunk 3]
```diff
@@ -29,6 +29,243 @@
...
+      "base": "ibc/9F28238BDB687A86681B30764BA1437E079C3565D04FDD7862055A83F7E5C630",
+      "name": "BadDog (Osmosis)",
+      "display": "baddog",
+      "symbol": "BADDOG.osmo",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "osmosis",
+            "base_denom": "transfer/channel-113/factory/chihuahua1x4q2vkrz4dfgd9hcw0p5m2f2nuv2uqmt9xr8k2/achihuahuawifhat",
```
Comment (JeremyParish69) [inline omniflixhub/assetlist.json:246]: This is not the correct base denom, it's an ibc/hash denom of some kind

File: omniflixhub/assetlist.json (hunk 1 above)
Comment (harish551) [reply omniflixhub/assetlist.json:157]: registered

File: omniflixhub/assetlist.json (hunk 2 above)
Comment (harish551) [reply omniflixhub/assetlist.json:202]: done

File: omniflixhub/assetlist.json (hunk 3 above)
Comment (harish551) [reply omniflixhub/assetlist.json:246]: that chihuhua tokenfactory coin osmosis 
this is the ibc transfer 
https://dev.mintscan.io/omniflix/txs/58B38182FE9A5C4D9EDC177001C92281FBD95630A45EB0DD867BB16300A95FC9

should we use `factory/chihuahua1x4q2vkrz4dfgd9hcw0p5m2f2nuv2uqmt9xr8k2/achihuahuawifhat` as base denom or including with chihuahua channel trace

I'm really confused about this

Comment (harish551) [reply omniflixhub/assetlist.json]: removed channel trace part 
only used `factory/chihuahua1x4q2vkrz4dfgd9hcw0p5m2f2nuv2uqmt9xr8k2/achihuahuawifhat`

File: omniflixhub/assetlist.json [hunk 4]
```diff
@@ -29,6 +29,243 @@
...
+      "base": "ibc/9F28238BDB687A86681B30764BA1437E079C3565D04FDD7862055A83F7E5C630",
+      "name": "BadDog (Osmosis)",
+      "display": "baddog",
+      "symbol": "BADDOG.osmo",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "osmosis",
+            "base_denom": "factory/chihuahua1x4q2vkrz4dfgd9hcw0p5m2f2nuv2uqmt9xr8k2/achihuahuawifhat",
```
Comment (JeremyParish69) [inline omniflixhub/assetlist.json:246]: The base denom is for BADDOG once transferred to Osmosis is: ibc/2FFE07C4B4EFC0DDA099A16C6AF3C9CCA653CC56077E87217A585D48794B0BC7

Comment (JeremyParish69) [reply omniflixhub/assetlist.json:246]: as per: https://github.com/cosmos/chain-registry/blob/820c602410c8126c5920269dff3b1290f64e09b0/osmosis/assetlist.json#L13283

Comment (harish551) [reply omniflixhub/assetlist.json]: okay understood now
updated correct base denom

**Review by JeremyParish69 — APPROVED** [review]

**Comment by harish551** [comment]
Hello @JeremyParish69 
I believe this is ready to merge now. let me know if any changes are required

**Pull request merged**